package.json

```diff
@@ -74,29 +74,29 @@
     "eslint": "^8.56.0",
```

Comment:
looks like we haven't used `enzyme` and `enzyme-adapter-react-16` for a very long time too.  can we remove those two?